src/frontend/Sider/Personoversikt/Ytelseoversikt/Ytelseoversikt.tsx

```diff
@@ -1,13 +1,20 @@
 import React, { useEffect, useState } from 'react';
 
-import { Alert } from '@navikt/ds-react';
+import { Alert, Heading } from '@navikt/ds-react';
 
 import YtelserTabell from './YtelserTabell';
 import { useApp } from '../../../context/AppContext';
 import DataViewer from '../../../komponenter/DataViewer';
 import { Registerytelser, registerYtelseTilTekst } from '../../../typer/registerytelser';
 import { byggHenterRessurs, Ressurs } from '../../../typer/ressurs';
-import { formaterIsoDatoTid } from '../../../utils/dato';
+import { formaterTilTekstligDato } from '../../../utils/dato';
+
+const formaterYtelsesHeader = (ytelser: Registerytelser) => {
+    const infotyper = ytelser.hentetInformasjon.map((info) => registerYtelseTilTekst[info.type]);
+    const sisteType = infotyper.pop();
+    const dato = formaterTilTekstligDato(ytelser.tidspunktHentet);
+    return 'Perioder med ' + infotyper.join(', ') + ' eller ' + sisteType + ' fra og med ' + dato;
```

Comment:
Og/eller, vet ikke hva som er riktig men virker som det er og i skissene, mulig dere har blitt enige om eller
```suggestion
    return 'Perioder med ' + infotyper.join(', ') + ' og ' + sisteType + ' fra og med ' + dato;
```

Reply:
Nyeste skissene bruker "eller" :)

[https://www.figma.com/design/7eVA9o9YdfQi7MRuzx9wWu/Personoversikt?node-id=1-16665&t=O1NEgtqHeBMm3jHO-4](https://github.com/navikt/tilleggsstonader-sak-frontend/pull/493)